.circleci/config.yml

```diff
@@ -0,0 +1,74 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+      - image: circleci/python:2.7.15
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "requirements.txt" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run:
+          name: install dependencies
+          command: |
+            sudo pip install -r requirements.txt
+            mkdir artifacts
+            cp sns_to_ses_mailer.py artifacts
+            cp -r /usr/local/lib/python2.7/site-packages/jinja2 artifacts/
+            cp -r /usr/local/lib/python2.7/site-packages/markupsafe artifacts/
+            cd artifacts
+            zip -r aws-sns-to-ses-mailer-0.0.1.zip *
+
+      - save_cache:
+          paths:
+            - /usr/local/lib/python2.7/site-packages/
+          key: v1-dependencies-{{ checksum "requirements.txt" }}
+
+      - store_artifacts:
+          path: aws-sns-to-ses-mailer-0.0.1.zip
+          destination: artifacts
+
+      - persist_to_workspace:
+          root: artifacts
+          paths:
+            - aws-sns-to-ses-mailer-0.0.1.zip
+  publish-github-release:
+      docker:
+        - image: cibuilds/github:0.10
+      steps:
+        - attach_workspace:
+            at: ./artifacts
+        - run:
+            name: "Publish Release on GitHub"
+            command: |
+              ls -la artifacts
+              VERSION="v0.0.1"
```

Comment:
I get that this is OK for now, but we probably need to find some version-bumping patterns for circleCI similar to the ones we have for concourse.